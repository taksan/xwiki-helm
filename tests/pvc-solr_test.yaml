suite: PersistentVolumeClaim (Solr)

templates:
  - pvc-solr.yaml

tests:
  - it: should properly set value for xwiki-solr pvc
    set:
      mysql.enabled: true
    asserts:
      - isKind:
          of: PersistentVolumeClaim
      - matchSnapshot:
          path: spec
      - equal:
          path: metadata.name
          value: xwiki-solr
