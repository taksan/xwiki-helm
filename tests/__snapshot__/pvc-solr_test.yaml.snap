should properly set value for xwiki-solr pvc:
  1: |
    accessModes:
    - ReadWriteOnce
    resources:
      requests:
        storage: 8Gi
